index.html

```diff
@@ -0,0 +1,132 @@
+     <li>Influenced by Mohandas Gandhi, he advocated civil disobedience and
+           nonviolent resistance to segregation in the South. The peaceful protests he led
+            throughout the American South were often met with violence,
+          but King and his followers persisted, and the movement gained momentum</li>
+     <li>In 1964, the civil rights movement achieved two of its greatest successes:
+            the ratification of the 24th Amendment, which abolished the poll tax,
+             and the Civil Rights Act of 1964, which prohibited racial discrimination in employment and
+              education and outlawed racial segregation in public facilities.</li>
+      <li>Later that year, King became the youngest person to win the Nobel
+            Peace Prize (in 2014 Malala Yousafzai became the youngest to receive the prize at age 17).</li>
+      <li>In the late 1960s, King openly criticized U.S. involvement in Vietnam and
+            turned his efforts to winning economic rights for poor Americans.
+             He was assassinated in Memphis, Tennessee, on April 4, 1968</li>
+    </ul>
+ </section>
+ <section id="pictures">
+   <h2> MLK Gallery</h2>
+   <div class="all">
+               <div class="pic">
+                   <img src="image/mlk1.jpg" alt="">
+               </div>
+               <div class="pic">
+                   <img src="image/mlk2.jpg" alt="">
+               </div>
+               <div class="pic">
+                   <img src="image/mlk3.jpg" alt="">
+               </div>
+               <div class="pic">
+                   <img src="image/mlk4.jpg" alt="">
+               </div>
+               <div class="pic">
+                   <img src="image/mlk5.jpg" alt="">
+               </div>
+               <div class="pic">
+                   <img src="image/mlk3.jpg" alt="">
+               </div>
+               <div class="pic">
+                   <img src="image/mlk4.jpg" alt="">
+               </div>
+                   <div class="pic">
+                       <img src="image/mlk1.jpg" alt="">
+                   </div>
+                   <div class="pic">
+                       <img src="image/mlk2.jpg" alt="">
+                   </div>
+                   <div class="pic">
+                       <img src="image/mlk3.jpg" alt="">
+                   </div>
+                   <div class="pic">
+                       <img src="image/mlk4.jpg" alt="">
+                   </div>
+                   <div class="pic">
+                       <img src="image/mlk5.jpg" alt="">
+                   </div>
+                   <div class="pic">
+                       <img src="image/mlk3.jpg" alt="">
+                   </div>
+                   <div class="pic">
+                       <img src="image/mlk4.jpg" alt="">
+                   </div>
+                   <div class="pic">
+                       <img src="image/mlk5.jpg" alt="">
+                   </div>
+               </div>
```

Comment:
Please kindly use <ul> and <li> tags in place of the divs for the images sections for all your images.👌

Reply:
changes implemented✔